pkg/xtrabackup/xtrabackup.go

```diff
@@ -546,6 +546,8 @@ func SetStoragePVC(job *batchv1.Job, pvc *corev1.PersistentVolumeClaim) error {
 func SetStorageS3(job *batchv1.Job, s3 *apiv1alpha1.BackupStorageS3Spec) error {
 	spec := &job.Spec.Template.Spec
 
+	bucket, _, _ := strings.Cut(s3.Bucket, "/")
```

Comment:
can we make this strings.Cut a function? we're using it in many places and with all these `_`, I believe the purpose is not clear.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/258/commits/c43f0b333323600668b3253d7f9fb6ff3eb1146c